test/test_versions.py

```diff
@@ -87,3 +98,54 @@ def test_api_versions_tuple_1(self):
             a = api_versions_tuple[i]
             b = api_versions_tuple[i+1]
             self.assertLess(a, b)
+
+    def test_version_invalid_new(self):
+        self.assertRaises(VersionError, Version, '1', 'a')
+        self.assertRaises(VersionError, Version, None, 0)
+        self.assertRaises(VersionError, Version, 1, 0, 0, 'final', None)
+        self.assertRaises(VersionError, Version, 1, 0, 0, 'invalid', 0)
+
+    def test_sortkey(self):
+        self.assertEqual((2, 1, 3, 4, 2), Version(2, 1, 3, 'final', 2).sortkey)
+        self.assertEqual((2, 0, 0, 1, 0), Version(2, 0, 0, 'a', 0).sortkey)
+        self.assertEqual((2, 0, 0, 1, 0), Version(2, 0, 0, 'alpha', 0).sortkey)
+
+    def test_lt(self):
+        v1 = Version(2, 3, 0, 'dev', 1)
+        v2 = Version(2, 3, 0, 'alpha', 1)
+        self.assertLess(v1, v2)
+        self.assertFalse(v2 < v2)
```

Comment:
What's the goal of this test?

Reply:
Verifying the __lt__ implementation compares properly